options.go

```diff
@@ -30,7 +30,7 @@ type Options struct {
 	AuthenticatedEmailsFile  string   `flag:"authenticated-emails-file" cfg:"authenticated_emails_file"`
 	AzureTenant              string   `flag:"azure-tenant" cfg:"azure_tenant"`
 	EmailDomains             []string `flag:"email-domain" cfg:"email_domains"`
-	GitHubOrg                string   `flag:"github-org" cfg:"github_org"`
+	GitHubOrgs               []string `flag:"github-orgs" cfg:"github_orgs"`
```

Comment:
I assume this is parsed as a comma-separated array automatically?

Reply:
It was! I'm starting to like Go (for other reasons as well).